src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1491,7 +1491,7 @@ private void SetupEditedSpecies(MicrobeSpecies species)
 
         // Disabled warning as this is a tweak constant
         // ReSharper disable ConditionIsAlwaysTrueOrFalse HeuristicUnreachableCode
-        if (Constants.CREATE_COPY_OF_EDITED_SPECIES)
+        if (new Random().NextDouble() < Constants.CREATE_COPY_OF_EDITED_SPECIES_CHANCE)
```

Comment:
Creating a new random here sounds wrong to me. This gets called multiple times at roughly the same time, right? The implementation of random bases it's seed upon the current time. If this gets called twice at the same time the random number will be the same.
Creating the random as an object variable fixes this.

Reply:
I'm pretty sure this gets called exactly once like 5 minutes (only once per editor object creation), so the current time as randomness seed is probably sufficient here.

Reply:
It's currently being called exactly twice each time the editor opens, since the helper method below also creates a random. I could combine the two, but I tend to shy away from passing down helper objects like that. I'm happy to code it either way.